test/v4/helper.js

```diff
@@ -101,44 +101,43 @@ module.exports.sendPromise = function(method, params) {
 ////////////////////////////////////////////////////////////////////////////////
 
 module.exports.exp = function(num1,num2) {
-    const num1Math = Math.bignumber(new BN(num1).toString(10));
-    const num2Math = Math.bignumber(new BN(num2).toString(10));
+    const num1Math = Math.bignumber(new BN(num1 * 10**9).toString(10)).div(10**9);
+    const num2Math = Math.bignumber(new BN(num2 * 10**9).toString(10)).div(10**9);
 
     const result = Math.pow(num1Math,num2Math);
 
-    return new BN(result.toString());
+    return result.toNumber();
 };
 
 module.exports.ln = function(num) {
-    const numMath = Math.bignumber(new BN(num).toString(10));
+    const numMath = Math.bignumber(new BN(num * 10**9).toString(10)).div(10**9);
```

Comment:
toString is required here, seems Math.bignumber can not recognise BN. we can use num * 10**9 directly but just in case someone passed BN inside this func